sdks/go/pkg/beam/transforms/sql/sql_test.go

```diff
@@ -0,0 +1,105 @@
+package sql
+
+import (
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/transforms/sql/sqlx"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+func TestOptions_Add(t *testing.T) {
+	o := options{}
+	str := "this is a string"
+	bytes := []byte{1, 2, 3, 4}
+	testOpt := sqlx.Option{
+		Urn:     str,
+		Payload: bytes,
+	}
+	tests := []struct {
+		opt sqlx.Option
+	}{
+		{
+			opt: testOpt,
+		},
+	}
+
+	for _, test := range tests {
+		o.Add(test.opt)
+		if o.customs == nil || !reflect.DeepEqual(o.customs[len(o.customs)-1], test.opt) {
+			t.Errorf("Add failed.")
+		}
+	}
+}
+
+func TestInput(t *testing.T) {
+	name1 := "this is a string"
+	collection1 := beam.PCollection{}
+	tests := []struct {
+		name string
+		in   beam.PCollection
+	}{
+		{
+			name: name1,
+			in:   collection1,
+		},
+	}
+
+	o := &options{inputs: make(map[string]beam.PCollection)}
+	for _, test := range tests {
+		option := Input(test.name, test.in)
+		if option == nil {
+			t.Errorf("%v return a nil value, which is not expected", runtime.FuncForPC(reflect.ValueOf(TestInput).Pointer()).Name())
+		}
+		option(o)
+		if o.inputs == nil || !reflect.DeepEqual(o.inputs[test.name], test.in) {
+			t.Errorf("%v did not modify inputs, which is not expected. Expected: %v. Got: %v", runtime.FuncForPC(reflect.ValueOf(TestExpansionAddr).Pointer()).Name(), test.in, o.inputs[test.name])
```

Comment:
Prefer "want" instead of "expected". It means the same thing and is half the size.

https://google.github.io/styleguide/go/decisions#got-before-want